template/web/src/components/header/AccountInfoPanel.tsx

```diff
@@ -27,6 +49,14 @@ export function AccountInfoPanel() {
         </div>
       </div>
       <hr className="h-px self-stretch border-transparent bg-zinc-400 bg-opacity-20" />
+      <div className="inline-flex flex-col py-4">
+        <span className="font-inter w-32 text-sm font-medium text-zinc-400">{chain?.name}</span>
+        <span className="font-inter w-32 text-base font-medium text-white">
+          {/* TODO: replace with OnchainKit component */}
+          {parseFloat(formatEther(balance)).toFixed(3)}{' ETH'}
```

Comment:
I think we should merge the onchain kit PR first and then merge this one, what do you think?

Reply:
cool